src/api/config/routes.rb

```diff
@@ -412,6 +406,16 @@ def self.public_or_about_path?(request)
       post 'users/announcements/:id' => :create, as: 'user_announcements'
     end
 
+    resources :users, only: [:index, :new, :create], controller: 'webui/users'
+
+    get 'signup', to: 'webui/users#new', as: :signup
+
+    # TODO
+    # keep those routes reachable, but remove them later as
+    # nobody access it anymore
+    get 'user/signup', to: redirect('/signup')
+    get 'user/register_user', to: redirect('/signup')
```

Comment:
This isn't working in the review app. It's a 404. However, it's working locally since we don't have anything following the domain name (unlike the review app with `vpereira-user_controller_bootstrap/` in this case)

https://obs-reviewlab.opensuse.org/vpereira-user_controller_bootstrap/user/register_user
https://obs-reviewlab.opensuse.org/vpereira-user_controller_bootstrap/user/signup

Is this a blocker? I don't think so, but let me know what you think.

Reply:
> Is this a blocker? 

Sure is, we support installing in a sub-directory.

Reply:
> Sure is, we support installing in a sub-directory.

I'm not sure it's a blocker @hennevogel we have it in another places like: 

https://github.com/openSUSE/open-build-service/blob/ef8fb3b79b43a1089a4edb860ec1e88ce78a0d4c/src/api/config/routes.rb#L323 

https://github.com/openSUSE/open-build-service/blob/ef8fb3b79b43a1089a4edb860ec1e88ce78a0d4c/src/api/config/routes.rb#L145

https://github.com/openSUSE/open-build-service/blob/ef8fb3b79b43a1089a4edb860ec1e88ce78a0d4c/src/api/config/routes.rb#L148

...

Reply:
Hm okay, then please record an issue about this.